src/hdmf/spec/catalog.py

```diff
@@ -56,7 +56,7 @@ def get_spec(self, **kwargs):
         return self.__specs.get(data_type, None)
 
     @docval(rtype=tuple)
-    def get_registered_types(self, **kwargs):
```

Comment:
Since this is a docval method, it might need this.

Reply:
ah, ok. i'll add it back.